articles/azure-monitor/app/source-map-support.md

```diff
@@ -0,0 +1,70 @@
+[create blob container]: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal
+[storage blob data reader]: https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#storage-blob-data-reader
+[ApplicationInsights-JS]: "https://github.com/microsoft/applicationinsights-js"
+[ApplicationInsights-Node.js]: "https://github.com/microsoft/applicationinsights-node.js"
+[azure file copy]: "https://aka.ms/azurefilecopyreadme"
+
+
+<!-- Local Images -->
+[unminify gif]: ./media/javascript/details_unminify.gif "Unminify a Call Stack by linking with a Storage Account"
+[container private]: ./media/javascript/container_access_level.png "Your container access level must be set to Private"
+[azure file copy image]: ./media/javascript/azure_file_copy.png "Add an Azure File Copy task to your Pipeline to upload your source maps to Azure Blob Storage"
+[reconfigure]: ./media/javascript/reconfigure.png "Reconfigure your selected Azure Blob Container by navigating to the Properties Blade"
+
+# Source Map Support for JavaScript Applications
+
+Application Insights supports the uploading of source maps to your own Storage Account Blob Container.
+Source maps can be used to unminify call stacks found on the End to End Transaction Details page. Any exception sent by the [JavaScript SDK][ApplicationInsights-JS] or the [Node.js SDK][ApplicationInsights-Node.js] can be unminified with source maps.
+
+![unminify gif]
+
+## Create a New Storage Account and Blob Container
+
+> *Note:* If you already have an existing storage account or blob container, you can skip this step.
+
+1. [Create a new storage account][create storage account]
+2. [Create a blob container][create blob container] inside your storage account. Be sure to set the "Public access level" to `Private` to ensure that your source maps are not publicly accessible.
+
+![container private]
+
+## Push your Source Maps to Your Blob Container
+
+Your should integrate your continuous deployment pipeline with your storage account by configuring it to automatically upload your source maps to the configured blob container. You should not upload your source maps to a subfolder in the Blob Container -- currently the source map will only be fetched from the root folder.
+
+### Upload Source Maps via Azure Pipelines (recommended)
+
+If you are using Azure Pipelines to continuously build and deploy your application, simply add an [Azure File Copy][azure file copy] task to your pipeline to automatically upload your source maps.
+
+![azure file copy image]
+
+## Configure Your Application Insights Resource with a Source Map Storage Account
+
+From the end-to-end transaction details blade, you can click on `Unminify` and it will automatically allow you to configure if you have not done so already.
+
+## Reconfigure Your Source Map Storage Account
```

Comment:
I'm wondering if we should frame it as Two ways to configure Source Map storage account and swap the order. I'd imagine most users will use settings as they won't have an active exception when reading the article.

Reply:
Good idea. Updated in a02c6c7

Reply:
Yes, much better. Is it possible to configure for the first time from the properties page? If yes, that's not clear to me. The way I interpret is that it's only possible to change or re-configure from properties.

Reply:
Updated in a02c6c7

Reply:
Awesome, looks great. TY.